tests/acceptance/features/webUISharingInternalGroups/shareWithGroups.feature

```diff
@@ -207,12 +227,14 @@ Feature: Sharing files and folders with internal groups
   @issue-2060
   Scenario: sharing indicator of items inside a re-shared folder
     Given user "user1" has shared folder "simple-folder" with user "user2"
-    And user "user2" has shared folder "simple-folder (2)" with group "grp1"
+    And user "user2" has accepted the share "simple-folder" offered by user "user1"
+    And user "user2" has shared folder "Shares/simple-folder" with group "grp1"
     When user "user2" has logged in using the webUI
+    And the user opens folder "Shares" using the webUI
     Then the following resources should have share indicators on the webUI
       | fileName            | expectedIndicators |
-      | simple-folder (2)   | user-direct        |
-    When the user opens folder "simple-folder (2)" using the webUI
+      | simple-folder   | user-direct        |
+    And the user opens folder "simple-folder" using the webUI
```

Comment:
```suggestion
    When the user opens folder "simple-folder" using the webUI
```

You used `When`for this step in other scenarios, so at least we should be consistent.

The exact flow of these kind of tests can be discussed separately from this PR.